### ISSUE TITLE
Italic

### ISSUE THREAD

**Issue by arjpar** [Corne2Plum3/Giphurs#28]
Currently there are only roman styles. It would be useful to have italic styles. Great work so far!

**Comment by Corne2Plum3** [comment]
It's planned.

**Comment by arjpar** [comment]
Are the italics planned to be true italics by any chance?

**Comment by Corne2Plum3** [comment]
> Are the italics planned to be true italics by any chance?

What do you mean exactly?

**Comment by arjpar** [comment]
True italics are like the italics in Segoe ui, inter, ibm plex sans. Obliques are just the Roman style slanted. So it would be the regular/bold styles just slanted. True italics are common in serifs.

**Comment by Corne2Plum3** [comment]
@arjpar Are you excepting something like this?

![image](https://github.com/user-attachments/assets/0e1099cb-d851-4865-aabd-f049d83116d5)

**Comment by arjpar** [comment]
@Corne2Plum3 Yup! Exactly! Specifically, something similar to IBM Plex Sans, but that's very much in the same line.

**Comment by Corne2Plum3** [comment]
You can try the italics here: https://github.com/Corne2Plum3/Giphurs/releases/tag/v1.9.9
